scripts/checkdeps

```diff
@@ -122,6 +122,17 @@ if [ -n "${DISTRO_DEPS_PKG}" ] ; then
   deps_pkg+=(${DISTRO_DEPS_PKG})
 fi
 
+# aarch64 dependencies
+if [ "${MACHINE_HARDWARE_NAME}" = "aarch64" ]; then
```

Comment:
scripts/checkdeps does not source config/options. It did until fairly recently, see https://github.com/LibreELEC/LibreELEC.tv/pull/3965/commits/db6d111ecf37c05376c3e05a493df31f2605dc60 so you'd need to go down that rabbit hole and fix that issue before being able to source config/options again. In this case we only call scripts/checkdeps once per build, so use $(uname -m) rather than try to source config/options.

What happens if/when the soname versions (.2, .6 etc) change?

Is building qemu-x86_64 and qemu-user-binfmt an option?

Reply:
I replaced with uname -m. I think it is clear that way considering that all the places checkdeps is already called already calls config/options before. As a result I have changed the commit.

Related to libc:
This is required by the rockchip utilities part of rkbin package. Considering that may not change very frequently I am not sure this will be an issue mid term.
Compiling libc for x86_64 always may be an overkill as we need to use an x86_64 gcc.